app/models/report.rb

```diff
@@ -28,23 +28,17 @@ class << self
 
   REPORT_FIELDS = [
     {
-      field: 'druid', label: 'Druid',
-      proc: ->(doc) { doc.druid },
+      field: SolrDocument::FIELD_BARE_DRUID, label: 'Druid',
       sort: true, default: true, width: 100, formatter: 'linkToArgo',
       category: FREQUENTLY_USED_CATEGORY
     },
     {
-      field: 'purl', label: 'Purl',
-      proc: ->(doc) { "#{Settings.purl_url}/#{doc.druid}" },
-      solr_fields: %w[id],
```

Comment:
solr_fields is no longer supported since each field corresponds with a single solr field.